packages/shared/routes/setup/Profile.svelte

```diff
@@ -119,7 +119,7 @@
             disabled={busy}
             submitHandler={handleContinueClick}
         />
-        {#if get(stage) === Stage.PROD}
+        {#if true}
```

Comment:
Why not get rid of the if statement entirely?

Reply:
Done